### PR TITLE
Creating a homepage of the CSS framework we created

### PR DESCRIPTION
**Pull request by jairjaramillo**
Me and my partner are working on a CSS Framework with Grids! It has a lot of features for containers, texts, positioning and it's also responsible!

Please share any suggestions of how to make it better. Thanks!

### CODE REVIEW

File: css/grid.css [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+.nav-item {
+	text-align: center;
+	margin-left: 10px;
+	margin-right: 10px;
+}
+
+.nav-menu {
+	display: none;
+}
```
Comment (stratospherique) [inline css/grid.css]: - unnecessary classe. :pray: delete it

File: css/grid.css [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+@media only screen and (max-width: 720px) {
+	.grid-md-1,
+	.grid-md-2,
+	.grid-md-3,
+	.grid-md-4,
+	.grid-md-5,
+	.grid-md-6,
+	.grid-md-7,
```
Comment (stratospherique) [inline css/grid.css]: `.grid-md` classes should go inside the media queries for medium screen width. :no_entry:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="css/style.css">
```
Comment (stratospherique) [inline index.html]: - I cannot locate the file _style.css_

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work :clap: . However, some requirements needs to be implemented in order for us to approve your project. check below :arrow_down: 

#### Grid

The CSS files define:

* [x] Classes to define columns sized from 1/12th of the row to 100% of the row
* [x] The defined columns should have breakpoints (example column-large-2)
  - You have already defined some of these classes. check code comments below :arrow_down:  for more details.

#### General requirements:

- Run your html through the validator and fix the errors.

## Stretch Requirements Checklist
- The following requirement are meant when implemented to enrich your framework and make it more professional.

### Box utilities

#### Margins

* [x] The CSS files define separate classes for margin top, bottom, left, right, all margins and auto
* [x] The margins should span from `0 rem` to `1 rem`
* [x] The margins should have breakpoints (e.g. margin-sm-top-1)

#### Padding

* [x] The CSS files define separate classes for padding top, bottom, left, right, all padding and auto
* [x] The padding should span from `0 rem` to `1 rem`
* [x] The padding should have breakpoints (example padding-lg-2)

### Positioning

#### Floats

* [x] The CSS files define separate classes for float left, right and clearing the floats

#### Positions

* [x] The CSS files define separate classes to change the position property of an element to relative, absolute or fixed

### Display

#### Display properties

The CSS files define:

* [x] A class to set the display property of an element to flex
* [x] A class to set the display property to hidden
* [x] A class to set the display property to none
* [x] The none class should have breakpoints (example display-md-none)

#### Flex properties

The CSS files define:

* [x] Classes to set vertical alignment as top, bottom or center
* [x] Classes to set horizontal alignment as left, right or center

### Colors

#### Background and text

The CSS files define:

* [x] At least ten different classes to set the background and text colors of an element

### Text

#### Utilities

The CSS files define:

* [x] Classes to horizontally align text to the left, right or center
* [x] At least five classes to set the font size of an element
* [x] Three classes to set the font weight of an element (bold, normal, medium)

#### Border

* [x] The CSS files define separate classes for border top, bottom, left, right, all border
* [x] The border should span from 0 to 1rem
* [x] The border should have classes for making box round
* [x] The border should have a class defined to transform a box into a circle

### That's it :relaxed: . You're close keep up :muscle: . When finished, kidnly resubmit another code review request :pray:

File: css/grid.css (hunk 2 above)
Comment (jairjaramillo) [reply css/grid.css]: These classes are already on the media query for the medium screen, as you can see from the line 384 to the line 392. The classes you're seeing here is to specify such classes will remain taking the whole 12 grids when the screen is small. These lines are necessary to make the classes work as intended, just like in Bootstrap.

**Review by wathigo — CHANGES_REQUESTED** [review]
## Great work @jairjy  and @digomes87  :clap:   
### The documentation and the cloned page looks great!
Before we mark this as complete, let's fix a few issues
**[REQUIRED]** 
* Your CSS and HTML code contains errors. Upload your code to [w3 validator](https://validator.w3.org/#validate_by_upload) and fix all the errors.
* All unimportant files such as `.DS_Store` should not be committed. You can avoid committing them by including them in `.gitignore` file.

**After implementing the changes, submit another request for review** 
_Happy coding :computer:_

**Review by wathigo — APPROVED** [review]
### Hello Devs :wave:  
You have put great effort on the project!
**This is approved!** 
_Good luck in your future projects!_

**Pull request merged**